### PR TITLE
Never Gonna Give You Preview

### LINKED ISSUE #84: BUG: Link with hidden preview are sent with preview
If a spot contain a link and and the preview was hidden before sending, the bot will send it with preview in any case.

### PR DESCRIPTION
**Pull request by drendog**
Close #84 

Help is needed in `tests/unit/test_bot.py:539` which the telegram simulator remove the `reply_to_message` after the message has been edited, so when the post is sent to the admin channel it will crash.

### CODE REVIEW

File: modules/utils/info_util.py [hunk 1]
```diff
@@ -240,7 +242,7 @@ def send_post_to_admins(self) -> bool:
         message = self.__message.reply_to_message
         group_id = Config.meme_get('group_id')
         poll = message.poll  # if the message is a poll, get its reference
-
+
```
Comment (Helias) [inline modules/utils/info_util.py:245]: I suggest you whitespace plugin for vscode to remove automatically useless spaces

**Comment by Helias** [comment]
pylint is complaining, fix lint please

**Comment by TendTo** [comment]
I made some adjustments to the tests.
The edited message will not have the *reply\_to\_message* field (it is something that seems related to the bot API).
So, instead, we call the callback query on the original message (the one that was then edited) with the callback data it will receive

**Review by TendTo — APPROVED** [review]

**Comment by drendog** [comment]
Thanks!!

**Review by TendTo — APPROVED** [review]

File: tests/unit/test_bot.py [hunk 2]
```diff
@@ -544,6 +534,21 @@ def test_spot_link_with_preview(self, telegram: TelegramSimulator):
...
             assert MessageEntity.URL in types
+        
+        def test_spot_link_without_preview(self, telegram: TelegramSimulator):
+            """Tests the /spot command.
+            Complete with no the spot without link preview conversation
+            """
+            self.test_spot_link(telegram)
+
+            telegram.send_callback_query(text="No")
+            assert telegram.last_message.text == "Sei sicuro di voler publicare questo post?"
```
Comment (Helias) [inline tests/unit/test_bot.py:545]: "pubblicare"

**Review by Helias — APPROVED** [review]

**Pull request merged**